modules/end-user-guide/partials/proc_using-a-badge-to-link-to-workspaces.adoc

```diff
@@ -14,14 +14,14 @@ Use this badge and link it to a {prod-short} instance to quickly open a develope
 
 .Procedure
 
-Add a link to your repository in the project `README` file. Using link:https://che.openshift.io[che.openshift.io] as an example {prod-short} host, and a GitHub repository:
+Add a link to your repository in the project `README` file. Using link:https://workspaces.openshift.com[workspaces.openshift.com] as an example {prod-short} host, and a GitHub repository:
 
 ----
-[![Contribute](https://www.eclipse.org/che/contribute.svg)](https://che.openshift.io/#https://github.com/org/repository)
+[![Contribute](https://www.eclipse.org/che/contribute.svg)](https://workspaces.openshift.com/f?url=https://github.com/org/repository)
```

Comment:
Does it make sense to update link to be CRW specific?

Reply:
Since the naming for the new service is not CRW specific, let's leave it community-friendly for now